zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/PublishNotification.java

```diff
@@ -1,98 +1,145 @@
 package com.github.onsdigital.zebedee.model.publishing;
 
-import com.github.davidcarboni.httpino.Endpoint;
-import com.github.davidcarboni.httpino.Host;
-import com.github.davidcarboni.httpino.Http;
-import com.github.davidcarboni.httpino.Response;
+import com.github.davidcarboni.httpino.*;
 import com.github.onsdigital.zebedee.configuration.Configuration;
 import com.github.onsdigital.zebedee.exceptions.BadRequestException;
 import com.github.onsdigital.zebedee.json.ContentDetail;
 import com.github.onsdigital.zebedee.json.EventType;
 import com.github.onsdigital.zebedee.model.Collection;
+import com.github.onsdigital.zebedee.model.publishing.cacheAPI.NotificationPayloadCacheApi;
+import com.github.onsdigital.zebedee.model.publishing.cacheAPI.SendNotificationCacheApi;
 import com.github.onsdigital.zebedee.util.SlackNotification;
 import com.github.onsdigital.zebedee.util.slack.PostMessageField;
 import org.joda.time.DateTime;
 
 import java.text.SimpleDateFormat;
+import java.util.ArrayList;
 import java.util.Date;
 import java.util.List;
 
-import static com.github.onsdigital.logging.v2.event.SimpleEvent.info;
 import static com.github.onsdigital.logging.v2.event.SimpleEvent.error;
+import static com.github.onsdigital.logging.v2.event.SimpleEvent.info;
+import static com.github.onsdigital.zebedee.model.publishing.cacheAPI.NotificationPayloadCacheApi.BULLETIN_SEGMENT;
 
 /**
  * Created by bren on 16/12/15.
  * Publish notification to be sent to the website for caching timing purposes
  */
+
 public class PublishNotification {
 
+    private NotificationPayload payload;
+    private final List<NotificationPayloadCacheApi> payloads = new ArrayList<>();
```

Comment:
Given that there is another variable called `payload`, this one should probably be renamed to `legacyCacheApiPayloads`, just for clarity.